resources/views/vendor/apidoc/partials/info.blade.php

```diff
@@ -192,7 +192,7 @@
 --------------|-------------------------------
 identify      | Allows reading of the public profile of the authorizing user (`/me`).
 friends.read  | Allows reading of the authorizing user's friend list.
-users.read    | Allows reading of public user profiles on behalf of the authorizing user.
+public        | Allows reading of public user profiles on behalf of the authorizing user.
```

Comment:
```suggestion
public        | Allows reading of publicly available data on behalf of the authorizing user.
```
`public` scope can access things that aren't necessarily tied to a user, like spotlights or beatmaps

Reply:
Thank you
committed